.github/workflows/integration-tests.yml

```
@@ -136,6 +136,10 @@ jobs:
               php_server
             }
 
+            route /* {
+              root * /symfony/src/Symfony/Component/HttpFoundation/Tests/Fixtures/response-functional/
+            }
+
```

Not sure if it's better, but this should be enough:

```suggestion
            root * /symfony/src/Symfony/Component/HttpFoundation/Tests/Fixtures/response-functional/
```

---

works for me as well 👍